api/server/router/image/image_routes.go

```diff
@@ -242,6 +242,8 @@ func (s *imageRouter) postImagesLoad(ctx context.Context, w http.ResponseWriter,
 	output := ioutils.NewWriteFlusher(w)
 	defer output.Close()
 	if err := s.backend.LoadImage(r.Body, output, quiet); err != nil {
+		w.WriteHeader(httputils.GetHTTPErrorStatusCode(err))
```

Comment:
Does this work? Would `LoadImage` have already written to the stream and thus already have sent a `200 OK`?
We could check `output.Flushed()`, although I would pretty much expect `LoadImage` to handle all things on the stream by this point.

Reply:
OK, can we gate this with a check for `!output.Flushed()`?

Reply:
Yes, I can add that